src/cmds/qstat.c

```diff
@@ -2606,7 +2606,7 @@ qstat -B [-f] [-F format] [-D delim] [ server_name... ]\n";
 	}
 	if (E_opt == 1 && mode == JOBS) {
 		/* allocate enough memory to store list of job ids */
-		job_list = calloc(argc-1,PBS_MAXCLTJOBID+1);
+		job_list = malloc((argc - 1) * (PBS_MAXCLTJOBID + 1));
```

Comment:
Come to think of it, when you calloc() the array, the 0th element will have a '\0' in it.  This won't happen if you malloc() the array.  Does this matter?

Reply:
It won't be an issue as we don't call this function without any job id's. But I've initialized it in my next commit to avoid any issues in the future.